docs/make.jl

```diff
@@ -38,20 +38,19 @@ copy_file("LICENSE.md",
           "\n" => "\n> ", r"^" => "# License\n\n> ")
 
 # Make documentation
-makedocs(sitename = "TrixiBase.jl",
+makedocs(modules = [TrixiBase],
+         sitename = "TrixiBase.jl",
          # Provide additional formatting options
          format = Documenter.HTML(
                                   # Disable pretty URLs during manual testing
                                   prettyurls = get(ENV, "CI", nothing) == "true",
                                   # Set canonical URL to GitHub pages URL
                                   canonical = "https://trixi-framework.github.io/TrixiBase.jl/stable"),
          # Explicitly specify documentation structure
-         pages = [
-             "Home" => "index.md",
-             "API reference" => "reference.md",
-             "Authors" => "authors.md",
-             "License" => "license.md",
-         ])
+         pages = ["Home" => "index.md",
+                  "API reference" => "reference.md",
+                  "Authors" => "authors.md",
+                  "License" => "license.md"])
```

Comment:
This change comes from the new formatter version. Should I also fix the JuliaFormatter version here?

Reply:
Yes, please 👍

Reply:
I fixed the JuliaFormatter version and used the format like it was before.